app/controllers/home_controller.rb

```diff
@@ -3,4 +3,9 @@ def index
     @products = Product.all
     @categories = Category.where(ancestry: nil)
   end
+
+  def search
+    @products = Product.where('lower(name) LIKE lower(?) OR lower(brand) LIKE lower(?)',
+                              "%#{params[:query]}%", "%#{params[:query]}%").where(status: 'active')
```

Comment:
```suggestion
    @products = Product.where('lower(name) LIKE lower(:term) OR lower(brand) LIKE lower(:term)',
                              term: "%#{params[:query]}%").active)
```

Uma dica, dá pra usar chaves nomeadas pra quando repetir termo aí fica legal que precisa escrever só uma vez 😄

Reply:
beleza